index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <script src="https://kit.fontawesome.com/1e7e3fccee.js" crossorigin="anonymous"></script>
+  <title>Aly Atalla</title>
+</head>
+<body>
+  <header>
+    <nav class="nav">
+        <div class="menu-button">
+            <i class="fa-solid fa-bars fa-xl"></i>
+        </div>
+        <div class="header-links">
+            <a class="header-links" href="#works">Experience</a>
+            <a class="header-links" href="#about">About</a>
+            <a class="header-links" href="#contact-me">Contact</a>
+        </div>
```

Comment:
kindly wrap this list of links in ul li tags this is important for semantic reasons